deployment/common/changeset/link_transfer_timelock_test.go

```diff
@@ -0,0 +1,121 @@
+package changeset_test
+
+import (
+	"context"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zapcore"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/common/changeset"
+	"github.com/smartcontractkit/chainlink/deployment/common/types"
+	"github.com/smartcontractkit/chainlink/deployment/environment/memory"
+)
+
+// TestLinkTransferTimelock tests the LinkTransferTimelock changeset by
+func TestLinkTransferTimelock(t *testing.T) {
```

Comment:
Probably need a separate test for the validation

Reply:
added validations tests here https://github.com/smartcontractkit/chainlink/pull/15512/commits/03e93827fffe21bbf491f371b84918ae965ffa7e